rero_ils/modules/notifications/templates/email/due_soon/eng.txt

```diff
@@ -4,13 +4,13 @@ Dear patron,
 
 The loan period of following documents is expiring:
 
-{%- for document in documents %}
+{%- for loan in loans %}
 
-Title : {{ document.title_text }} / {{ document.responsibility_statement }}
-Due date: {{ document.end_date }}
+Title : {{ loan.document.title_text }}
```

Comment:
Be sure that do do not you forget `responsibility_statement`.

Reply:
using the `title_format_text_head` method into `DocumentNotificationDumper`, the statement are added to the title_text.